### PR TITLE
Benefit additional info header

### LINKED ISSUE #929: Benefit Card headers should be able to display warnings/notifications (not just 'parent' benefits)
say for Disability Award. put that text in the (!) area. Check with Design for content

### PR DESCRIPTION
**Pull request by thomascraig**
resolves #929

### CODE REVIEW

File: components/benefit_cards.js [hunk 1]
```diff
@@ -11,11 +11,12 @@ import NeedTag from "./need_tag";
 import { css } from "react-emotion";
 import CardFooter from "./card_footer";
 import BenefitCardHeader from "./benefit_card_header";
+import BenefitCardHeaderMoreInfo from "./benefit_card_additional_info";
 
 const button = css`
   background-color: #3e57e2 !important;
   color: white !important;
-  text-align: right !important;
+  benefit_card_additional_infotext-align: right !important;
```
Comment (sastels) [inline components/benefit_cards.js]: I think this is a paste where it shouldn't be?

File: components/benefit_card_additional_info.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  transform: scale(0.9);
+  color: #434343;
+`;
+const headerDesc = css`
+  flex-grow: 1;
+  color: #434343;
+`;
+
+export class BenefitCardHeaderMoreInfo extends Component {
+  logExit = url => {
```
Comment (sastels) [inline components/benefit_card_additional_info.js]: you don't need the `logExit` function here, it's just used to log when the user clicks on a VAC site url in the other header.

**Review by sastels — APPROVED** [review]
Minor comments - awesome work! 🦑

File: components/benefit_card_additional_info.js (hunk 2 above)
Comment (thomascraig) [reply components/benefit_card_additional_info.js]: @sastels just seeing these comments now, were the files updated?

**Pull request merged**